converters/toZigbee.js

```diff
@@ -5263,6 +5263,35 @@ const converters = {
             return {state: {keypad_lockout: value}};
         },
     },
+    moes_105z_dimmer: {
+        key: ['state', 'percentage'],
```

Comment:
Please change `percentage` to `brightness` and instead of being a value from `0` to `100` handle values from `0` to `254`.

Reply:
Changed to brightness